build.yaml

```diff
@@ -266,7 +271,19 @@ common_cli:
     # defintion, and feed it to the job configuration as the default value for
     # the 'MODULE' parameter.
     # This is how we tell trial which flocker module to call.
-    coverage run \$venv/bin/trial --reporter=subunit \$MODULE 2>&1 | parse_logs tee trial.log
+    coverage run \$venv/bin/trial \
+      --reporter=subunit \$MODULE 2>&1 | parse_logs tee trial.log
+
+  run_trial_with_coverage_as_root: &run_trial_with_coverage_as_root |
+    # The jobs.groovy.j2 file produces jobs that contain a parameterized job
+    # type. These type of jobs always require a parameter to be passed on in
+    # order for they to be executed.
+    # We grab the value from the 'with_modules:' dictionary in the yaml job
+    # defintion, and feed it to the job configuration as the default value for
+    # the 'MODULE' parameter.
+    # This is how we tell trial which flocker module to call.
+    sudo \$venv/bin/coverage run \$venv/bin/trial \
+      --reporter=subunit \$MODULE 2>&1 | parse_logs tee trial.log
```

Comment:
:question: The only difference between this command and the one bellow is that `$MODULE` there is prefixed by `--test-module`. It seems like maybe this difference could be lifted to a different level.
1. These are invoked by different groovy job types and the difference could be handled there.
2. Those two groovy job types could be combined by having one job that looks at two different yaml keys, one of which gets its values prepended with `--test-module`.